lib/core-integration/src/Test/Integration/Scenario/CLI/Shelley/Transactions.hs

```diff
@@ -839,7 +840,112 @@ spec = do
               oJson2 <- expectValidJSON (Proxy @([ApiTransaction n])) o2
               length <$> [oJson1, oJson2] `shouldSatisfy` all (== 0)
 
-    it "TRANS_DELETE_01 - Cannot forget pending transaction when not pending anymorevia CLI" $ \ctx -> do
+    it "TRANS_GET_01 - Can get Incoming and Outgoing transaction" $ \ctx -> do
```

Comment:
Same comment than for the API scenario applies here too :+1:

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1795/commits/d898b0d84bba687166dcd71bfd440d29cd26f063